pom.xml

```diff
@@ -59,6 +71,17 @@
             <version>5.8.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.openjfx</groupId>
+            <artifactId>javafx-base</artifactId>
+            <version>16</version>
+        </dependency>
```

Comment:
Ak sa použije Java 8, tak toto by asi nemalo byť potrebné.

Reply:
Funguje ti bez
`<groupId>org.openjfx</groupId>
            <artifactId>javafx-base</artifactId>
            <version>16</version>`
FitnessApplicationView? Lebo mi sa to vsetko rozsvetilo cervenym

Reply:
Máš prepnutú Javu na verziu 8?

Reply:
ano, mam

Reply:
Pushni zmeny nech to skusim rozbehať u seba.

Reply:
okej

Reply:
hotovo